source/builtin/Cargo.toml

```diff
@@ -2,6 +2,14 @@
 name = "verus_builtin"
 version = "0.1.0"
 edition = "2018"
+license = "MIT"
+description = "Verus Builtins: Rust definitions for core constructs in the Verus language"
```

Comment:
I would say "in Verus" rather than "in the Verus language"

Reply:
Done.